contributing/STEERING-COMMITTEE.md

```diff
@@ -5,65 +5,156 @@ weight: 40
 
 - [Charter](#charter)
+- [Delegated authority](#delegated-authority)
+- [Committee Meetings](#committee-meetings)
 - [Committee Mechanics](#committee-mechanics)
 - [Committee Members](#committee-members)
+  - [Allocation of seats](#allocation-of-seats)
+- [Decision process](#decision-process)
+- [Changes to the charter](#changes-to-the-charter)
+- [Getting in touch](#getting-in-touch)
 
 ## Charter
 
-- Non-technical project oversight
-
-  - Define policy for the creation and administration of community groups,
-    including [Working Groups](./WORKING-GROUPS.md) and Committees.
-
-  - Define and evolve project governance structures and policies, including
-    project role assignment and contributor promotion.
-
-  - Approve members of the Tech Oversight Committee.
-
-- Management of project assets
-
-  - Control access to, establish processes regarding, and provide a final
-    escalation path for any Knative repository.
+1. Define, evolve, and promote the vision, values, mission, and scope of the project.
+1. Define and evolve project governance structures and policies, including
+   project roles and how collaborators become members, approvers, leads,
+   and/or administrators. This includes policy for the creation and
+   administration of [working groups](./WORKING-GROUPS.md) and committees.
+1. Steward, control access, delegate access, and establishes processes regarding,
+   all Knative project resources and has the final say in the disposition of
+   those resources.
+1. Manage the Knative brand and decide which things can be called "Knative" and
+   how that mark can be used in relation to other efforts or vendors.
+1. Confirm/reject nominations to the KSC from organizations who are allocated seats.
+1. Confirm/reject nominations to the Technical Oversight Committee.
+1. Receive and handle reports about [code of conduct](./CODE-OF-CONDUCT.md)
+   violations and maintain confidentiality.
+1. Receive security reports; work with the appropriate technical leads to
+   accept or reject the report; maintain the private nature of such reports
+   until disclosed to the broader community.
+1. Act as the final escalation point and decider for any disputes, issues,
+   clarifications, or escalations within the project scope.
+
+## Delegated authority
+
+KSC may choose to delegate its authority to other committees as-needed. The
+committee currently recognizes this delegated authority for:
+
+- Technical guidance is delegated to the [Technical Oversight Committee](./TECH-OVERSIGHT-COMMITTEE.md).
+
+## Committee Meetings
+
+KSC meets every two weeks, or as-needed. Meetings are held online.
+
+Given the private nature of many of these discussions (e.g. privacy, private
+emails to the committee, code of conduct violations, escalations, disputes
+between members, security reports, etc.) meetings are held in private.
+
+Meeting notes are available to members of the knative-dev mailing list
```

Comment:
If all meetings are private then having public minutes sees odd. To go along with my comment above, I would make the public meetings have public notes, but then private meetings can have private/secret notes only available to SC members.